deploy/examples/monitoring/grafana/README.md

```diff
@@ -0,0 +1,13 @@
+# Grafana Dashboards
+
+This folder contains the JSON files for the Grafana dashboards.
+The dashboards are based upon [the official Ceph Grafana dashboards](https://github.com/ceph/ceph/tree/main/monitoring/ceph-mixin) but with some slight tweaks.
+
+## Updating the Dashboards
+
+To update the dashboards, please export them via Grafana's export function.
+
+!!! note
```

Comment:
Since this readme isn't in the Documentation folder, notes like this aren't rendered, it's just the basic markup rendering. Shall we just remove this line?

Reply:
Ah yeah, fixed it